libs/core/src/lib/list/list-focus-item.model.ts

```diff
@@ -0,0 +1,23 @@
+import { ElementRef, EventEmitter } from '@angular/core';
+
+import { KeyboardSupportItemInterface } from '../utils/public_api';
+
+export class ListFocusItem implements KeyboardSupportItemInterface {
+    /** @hidden Implementation of KeyboardSupportItemInterface */
+    keyDown: EventEmitter<KeyboardEvent> = new EventEmitter<KeyboardEvent>();
+
+    constructor (
+      readonly elementRef: ElementRef
+    ) {
+    }
+
+    /** @hidden */
+    click(): void {
+      this.elementRef.nativeElement.click();
```

Comment:
this class a chance to exists before or native element exists
```suggestion
      this.elementRef?.nativeElement?.click();
```

Reply:
Could you please reproduce situation when component/directive constructor have no elementRef?